### PR TITLE
Initialized Home Page(STARTER)

### PR DESCRIPTION
**Pull request by FishLegs**
Added in items, and basic components of a home page, we can decide on style of home page soon.

### CODE REVIEW

**Comment by FishLegs** [comment]
Screenshot 2024-11-08 at 8 16 08 AM

**Review by rferrall1738 — APPROVED** [review]
Add date found as part of the tags of the item. Sort items by the most recent find

File: packages/react-frontend/src/HomePage.jsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+const items = [
+  { id: 1, title: "Lost Wallet", location: "Library", description: "Black leather wallet" },
+  { id: 2, title: "Found Keychain", location: "Dining Hall", description: "Red keychain with keys" },
+  { id: 3, title: "Lost Backpack", location: "Engineering Building", description: "Blue backpack with laptop" },
+  { id: 4, title: "Found Water Bottle", location: "Gym", description: "Green water bottle with stickers" },
+  { id: 5, title: "Lost Headphones", location: "Student Center", description: "Wireless headphones in a black case" },
+  { id: 6, title: "Found Sunglasses", location: "Parking Lot C", description: "Black sunglasses with polarized lenses" },
+  { id: 7, title: "Lost Notebook", location: "Coffee Shop", description: "Red notebook with math notes" },
+  { id: 8, title: "Found Calculator", location: "Math Lab", description: "Graphing calculator with initials" },
+];
```
Comment (JacobLilien) [inline packages/react-frontend/src/HomePage.jsx:12]: Might want to seperate the lost items from the found items

**Review by JacobLilien — COMMENTED** [review]
Style of the new page looks great

File: packages/react-frontend/src/HomePage.jsx (hunk 1 above)
Comment (FishLegs) [reply packages/react-frontend/src/HomePage.jsx:12]: for right now I think its best to leave it as random. But in a new issue I can handle for lost/found and sort them.

**Pull request merged**